terraform/main.tf

```diff
@@ -172,6 +167,14 @@ variable "victorops_routing_key" {
   description = "VictorOps/Splunk OnCall routing key for prometheus-alertmanager"
   default     = "bogus-routing-key"
 }
+variable "cluster_settings" {
+  type = object({
+    initial_node_count : number
```

Comment:
I didn't know this was also valid syntax for TF object definitions! Elsewhere we use `name = type`, and if possible I'd prefer to use that here for consistency.

Reply:
I actually got this syntax for where we define ingestors: https://github.com/abetterinternet/prio-server/blob/main/terraform/main.tf#L36-L45

Reply:
Oh now I see what you meant, using `=` instead of `:`

I'm really curious if there is a difference between these two.